internal/signer/darwin/signer.go

```diff
@@ -32,17 +32,34 @@ import (
 	"github.com/googleapis/enterprise-certificate-proxy/internal/signer/util"
 )
 
+
 // If ECP Logging is enabled return true
 // Otherwise return false
-func enableECPLogging() bool {
+func enabledECPLogging() bool {
```

Comment:
We should move all logging related helpers to a shared util file if possible so it's not duplicated.